### PR TITLE
RSDK-7751 - Implement new SetRPM for Motor API

### PR DESCRIPTION
**Pull request by gvaradarajan**
NOTE: Hardware testing incomplete. Will not merge until then, but would appreciate a quick code review

### CODE REVIEW

File: micro-rdk/src/common/motor.rs [hunk 1]
```diff
@@ -106,6 +106,13 @@ pub trait Motor: Status + Actuator + DoCommand {
     /// If revolutions is 0, this will run the motor at rpm indefinitely.
     /// If revolutions != 0, this will block until the number of revolutions has been completed or another operation comes in.
     fn go_for(&mut self, rpm: f64, revolutions: f64) -> Result<Option<Duration>, MotorError>;
+    // Instructs the motor to turn at the specified RPM. The default behavior is to call go_for
```
Comment (acmorrow) [inline micro-rdk/src/common/motor.rs:109]: Personal preference nit, feel free to ignore: could you throw blank lines in this trait declaration before starting a new comment block?

**Review by acmorrow — APPROVED** [review]
LGTM

File: micro-rdk/src/common/grpc.rs [hunk 2]
```diff
@@ -32,7 +32,7 @@ use std::rc::Rc;
 use std::task::{Context, Poll};
 use thiserror::Error;
 
-use super::webrtc::grpc::WebRtcGrpcService;
+use super::{motor::Motor, webrtc::grpc::WebRtcGrpcService};
```
Comment (mattjperez) [inline micro-rdk/src/common/grpc.rs:35]: nit: move this up to the imports at L10. Could probably move the `webrtc` import there too if no one else minds a little code movement.

File: micro-rdk/src/common/grpc.rs [hunk 3]
```diff
@@ -234,6 +234,7 @@ where
             }
             "/viam.component.motor.v1.MotorService/SetPower" => self.motor_set_power(payload),
             "/viam.component.motor.v1.MotorService/Stop" => self.motor_stop(payload),
+            "/viam.component.motor.v1.MotorService/SetRPM" => self.motor_set_rpm(payload),
```
Comment (mattjperez) [inline micro-rdk/src/common/grpc.rs:237]: nit: move a line up above `motor_stop`

**Review by mattjperez — APPROVED** [review]
LGTM!
couple non-blocking nits

**Comment by gvaradarajan** [comment]
Hardware testing complete (using Python SDK). Merge incoming

**Pull request merged**